src/Raven.Client/Json/Converters/BlittableJsonConverter.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Sparrow.Json;
+
+namespace Raven.Client.Json.Converters
+{
+
+    internal sealed class BlittableJsonConverter : RavenJsonConverter
+    {
+        public static readonly BlittableJsonConverter Instance = new BlittableJsonConverter();
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            writer.WriteValue(value);
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            if (!(reader is BlittableJsonReader blittableReader))
+            {
+                throw new SerializationException(
```

Comment:
It's better to move the throws to seperate functions

Reply:
I move the check to a base class and I think now it's better.
What do you think?